.gitlab-ci.yml

```diff
@@ -580,6 +580,30 @@ chaos-test-singlenodeheight:
   after_script:
     - chaostest clean
 
+
+# job: chaos run weekly
+chaos-weekly:
+  environment:
+    name:                          parity-chaosnet
+  image:                           parity/chaostools:latest
+  interruptible: true
+  needs:
+    - job:                         docker-build-chaos
```

Comment:
i'm not 100%ly sure if this will build automatically. I assume you would need to extend the rules of the depending jobs as well.

Reply:
no, it will not. The good way to run something on a schedule is to set a cron [scheduled pipeline](https://gitlab.parity.io/parity/substrate/-/pipeline_schedules) with a variable. And set this variable to `rules:` of the dependent jobs. Like this:
```
rules:
  - if: '$YOUR_VARIABLE == "your_value"'
    when: always
  - if: '$CI_PIPELINE_SOURCE == "schedule"'
  - # other rules
```
This approach wouldn't even require an additional job.

Reply:
suggest to add a "- when: never " as default rule

Reply:
The idea of adding the additional variable here is that otherwise this job would run on any schedule. There is no other way to select specific jobs for specific schedules.

Reply:
actually, I have to delete this pull request; found an easier alternative with a k8s cronjob